js/tests/views_calendarview_spec.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @author Steffen Lindner <mail@steffen-lindner.de>
+ *
+ * Mail
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+
+define(['views/calendarview', 'views/helper'], function(CalendarView) {
+
+	describe('CalendarView', function () {
+
+		var calendarview;
+
+		beforeEach(function () {
+			calendarview = new CalendarView({});
+		});
+
+		describe('Rendering', function () {
```

Comment:
do we really need the second `describe` block? I guess one *describe* and one *it* should be enough :wink: